src/tooltip/test/tooltip.spec.js

```diff
@@ -249,10 +249,11 @@ describe('tooltip', function() {
       scope.$digest();
     }));
 
-    it('should open after timeout', inject(function ($timeout) {
+    it('should open after timeout and set the close delay to the default', inject(function ($timeout) {
 
       elm.trigger('mouseenter');
       expect(tooltipScope.isOpen).toBe(false);
```

Comment:
I am directly testing the opening and closing. The ```isOpen``` variable is bound to the scope and is the main condition that opens and closes the tooltip. It is a boolean, can't get really get any different value other than: true, false, null, undefined.